library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/PojoFullOuterJoin.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.malhar.lib.window.accumulation;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Full outer join Accumulation for Pojo Streams.
+ *
+ */
+public class PojoFullOuterJoin<InputT1, InputT2>
+    extends AbstractPojoJoin
+{
+  public PojoFullOuterJoin()
+  {
+   super();
+  }
+
+  public PojoFullOuterJoin(int num, Class<?> outClass, String... keys)
+  {
+    super(outClass,keys);
+  }
+
+  @Override
+  public void addNonMatchingResult(List result, Map requiredMap, Set nullFields)
+  {
+    for (Object field : nullFields) {
+      if (!keySetStream2.contains(field)) {
+        requiredMap.put(field.toString(), null);
+      }
+    }
+    result.add(requiredMap);
+  }
+
+  @Override
+  public int getLeftStreamIndex()
+  {
+    return 0;
+  }
+
+  @Override
+  public Map<String, Object> joinTwoMapsWithKeys(Map map1, Map map2)
```

Comment:
This method seems to be doing the same thing as the one present in super class. Remove this?